home-application/src/main/java/com/softserveinc/ita/homeproject/config/HomeUserPrincipal.java

```diff
@@ -0,0 +1,64 @@
+package com.softserveinc.ita.homeproject.config;
+
+import com.softserveinc.ita.homeproject.model.Permission;
+import com.softserveinc.ita.homeproject.model.Role;
+import com.softserveinc.ita.homeproject.model.User;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Set;
+
+
+public class HomeUserPrincipal implements UserDetails {
```

Comment:
Please highlight in the class name that it is actually a wrapper for the actual User entity. It would be nice to provide some java doc.